aurora-rust-sdk/aurora-sdk-integration-tests/src/utils/forge.rs

```diff
@@ -52,10 +52,22 @@ pub async fn forge_build<P: AsRef<Path>>(
     root_path: P,
     libraries: &[String],
     contract_output_path: &[&str],
+) -> anyhow::Result<ContractConstructor> {
+    forge_build_with_args(root_path, libraries, contract_output_path, &[]).await
+}
+
+pub async fn forge_build_with_args<P: AsRef<Path>>(
+    root_path: P,
+    libraries: &[String],
+    contract_output_path: &[&str],
+    extra_args: &[&str],
 ) -> anyhow::Result<ContractConstructor> {
     let _guard = FORGE_LOCK.lock().await;
     let contracts_path = root_path.as_ref();
-    let args = std::iter::once("build").chain(libraries.iter().flat_map(|x| ["--libraries", x]));
+    let args = std::iter::once("build")
+        .chain(libraries.iter().flat_map(|x| ["--libraries", x]))
+        .chain(extra_args.iter().copied());
```

Comment:
I guess it would be good to encapsulate this logic into a separate function and add a unit test for it.